empyrical/stats.py

```diff
@@ -1005,6 +1005,46 @@ def cagr(returns, period=DAILY, annualization=None):
     return ending_value ** (1. / no_years) - 1
 
 
+def value_at_risk(returns, cutoff=0.05):
+    """
+    Parameters
+    ----------
+    returns : pandas.Series or 1-D numpy.array
+        Non-cumulative daily returns.
+    cutoff : float, optional
+        Decimal representing the percentage cutoff for the bottom percentile of
+        returns. Defaults to 0.05.
+
+    Returns
+    -------
+    VaR : float
+        The VaR value.
+    """
+    return np.percentile(returns, 100 * cutoff)
+
+
+def conditional_value_at_risk(returns, cutoff=0.05):
+    """
+    Parameters
+    ----------
+    returns : pandas.Series or 1-D numpy.array
+        Non-cumulative daily returns.
+    cutoff : float, optional
+        Decimal representing the percentage cutoff for the bottom percentile of
+        returns. Defaults to 0.05.
+
+    Returns
+    -------
+    CVaR : float
+        The CVaR value.
+    """
+    # PERF: Instead of using the 'value_at_risk' function to find the cutoff
+    # value, which requires a call to numpy.percentile, determine the cutoff
+    # index manually and partition out the lowest returns values.
+    num_low_days = int((len(returns) - 1) * cutoff) + 1
```

Comment:
The only simpler way I can think of is to rename `num_low_days` to something like `index_of_cutoff`. That way I don't have to do `+1` here, or `-1` below. Does that seem better/less confusing?